lib/modules/datasource/python-version/__fixtures__/release.json

```diff
@@ -0,0 +1,34 @@
+[
```

Comment:
please reduce to a few versions

Reply:
please reduce further for testing we only need 2 or 3 versions

Reply:
Reduced, leaving it at that. I'm testing pre-releases and deprecated.